bftengine/src/preprocessor/messages/PreProcessRequestMsg.hpp

```diff
@@ -89,7 +92,8 @@ class PreProcessRequestMsg : public MessageBase {
                  uint32_t spanContextSize,
                  uint64_t reqRetryId,
                  uint32_t reqLength,
-                 uint16_t reqSignatureLength);
+                 uint16_t reqSignatureLength,
+                 uint64_t block_id);
```

Comment:
->blockId

Reply:
too many conventions in our code base :)

Reply:
Yeah, I just want to preserve the convention existing in this file.

Reply:
sure, you're rught